scripts/ci/pack_soljson.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+SCRIPT_DIR="$(realpath "$(dirname "$0")")"
+SOLJSON_JS="$1"
+SOLJSON_WASM="$2"
+SOLJSON_WASM_SIZE=$(wc -c "${SOLJSON_WASM}" | cut -d ' ' -f 1)
+OUTPUT="$3"
+
+{
+  echo -n "\"use strict\";"
+  echo -n "var Module = Module || {}; Module[\"wasmBinary\"] = "
+  cat "${SCRIPT_DIR}/wasm_unpack.js"
+  echo -n "(\""
+  lz4c --no-frame-crc --best --favor-decSpeed "${SOLJSON_WASM}" - | base64 -w 0 | sed 's/[^A-Za-z0-9\+\/=]//g'
+  echo -n "\",${SOLJSON_WASM_SIZE});"
+  sed -e 's/"use strict";//' "${SOLJSON_JS}"
```

Comment:
The chance of a false-positive here is not low enough IMO. Maybe at least anchor it to the beginning of the string?
```suggestion
  sed -e 's/^"use strict";//' "${SOLJSON_JS}"
```

Reply:
Fair enough, yeah.

Reply:
Funnily, if I see it correctly, emscripten doesn't even emit ``"use strict";`` anymore these days... instead I'm seeing ``null;``... I guess that works similarly, but haven't found it documented so far - not an expert for all the js magic myself...

Reply:
In that case maybe it would be a good idea to also add a simple check that either `"use strict";` or `null;` is there? `grep` one-liner should do it (I think that `sed` does not use the exit code to indicate whether the replacement actually happened, unfortunately). If emscripten changes things again, we'll want to adjust this code to match. We need a way to discover that.